net/Socket.cpp

```diff
@@ -426,6 +426,7 @@ int SocketPoll::poll(int64_t timeoutMaxMicroS)
 
         // We use the _pollStartIndex to start the polling at a different index each time. Do some
         // sanity check first to handle the case where we removed one or several sockets last time.
+        ++_pollStartIndex;
```

Comment:
I'm missing a modulus here. This looks like it increments until it hits size-1 and stays there. Is that expected ?